interfaces/oauth_client/v0/README.md

```diff
@@ -0,0 +1,86 @@
+# `oauth_client`
+
+## Usage
```

Comment:
Please add an Overview heading and provide a brief (1 paragraph max, 1 sentence ideally) overview of the relation. This will be required for all interfaces going forward.

Reply:
Is the `Usage` section still required? It feels like the `Overview` section was included in the `Usage` section so far.

Reply:
Yes, I think they serve different purposes. Usage section is developer documentation on how to use the interface. Think of overview as the "executive summary".